tools/dotcms-cli/cli/src/test/java/com/dotcms/cli/command/site/SiteCommandIT.java

```diff
@@ -30,6 +30,7 @@
 import javax.inject.Inject;
 import javax.ws.rs.NotFoundException;
 import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.jetbrains.annotations.NotNull;
```

Comment:
I think this should go away we don't do jetbrains

Reply:
Done